scalableminds/webknossos
# dont throw initialization error when logged out

This PR removes the "Initialization Error. ..." toast that popped up when trying to view a dataset without being logged in.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- copy a link to a dataset view
- don't log in / log out
- go to the copied link
- you should be moved to the login page with an error toast that says you can not access the dataset
- there should be no "Initialization Error. ..." toast.

### Issues:
- fixes #4731 

------
